ingestion_tools/dataset_configs/10008.yaml

```diff
@@ -192,7 +192,7 @@ tiltseries:
       energy_filter: GIF Quantum LS
       phase_plate: null
     pixel_spacing: 3.52
-    related_empiar_entry: EMD-15053
```

Comment:
@uermel is there a reason we want this to capture just the EMPIAR values for tiltseries and not include the EMD entries?